BenadrylAutomation/src/main/java/com/jnj/pageobjects/ProductPage.java

```diff
@@ -0,0 +1,284 @@
+	 * @param expectedText The expected text to be displayed on the 'Buy Now' button.
+	 * @param expectedModalText The text expected to be found on the heading of the modal.
+	 */
+	public void verifyBuyNowBtn(String expectedBtnText, String expectedModalText) {
+	    this.visitPDP();
+	    extentInfoLog("Button is displayed : ", Action.isDisplayed(driver, buyNowBtn));
+	    String actualBtnText = buyNowBtn.getText();
+	    Action.printAndAssert(actualBtnText, expectedBtnText);
+	    Action.explicitWaitForElementTobeclickable(buyNowBtn, 30);
+	    Action.performActionwithExtentInfoLog(buyNowBtn, "click", "Clicking on : " + actualBtnText);
+	    Action.explicitWaitForElementTobeclickable(buyNowModalHeading, 30);
+	    String actualModalText = buyNowModalHeading.getText();
+	    Action.printAndAssert(actualModalText, expectedModalText);
+	    extentInfoLog("Buy Now modal is displayed : ", Action.isDisplayed(driver, buyNowModal));
+	}
+	
+	/**
+	 * Verifies the jump to section by clicking on the 'Jump To' heading and validating the resulting section heading.
+	 *
+	 * @param jumpToHeading The 'Jump To' heading to interact with
+	 * @param sectionHeading The expected heading text to verify afterwards
+	 */
+	public void verifyJumpToSection(String jumpToHeading, String sectionHeading) {
+		this.visitPDP();
+	    WebElement jumpToHead = getJumpToHeadings(jumpToHeading);
+	    String jumpToHeadingText = jumpToHead.getText();
+	    WebElement actualHeadingElement;
+
+		Action.scrollUntilElementVisible(jumpToTitle);
+	    Action.performActionwithExtentInfoLog(jumpToHead, "click", "Clicking on : " + jumpToHeadingText);
+
+	    switch (jumpToHeading.toLowerCase()) {
+	        case "overview":
+	            actualHeadingElement = prodOverviewHeading;
+	            break;
+	        case "directions":
+	            actualHeadingElement = directionHeading;
+	            break;
+	        case "ingredients":
+	            actualHeadingElement = ingredientsHeading;
+	            break;
+	        case "used for":
+	            actualHeadingElement = usedForHeading;
+	            break;
+	        case "warnings":
+	            actualHeadingElement = warningsHeading;
+	            break;
+	        case "faqs":
+	            actualHeadingElement = faqsHeading;
+	            break;
+	        case "reviews":
+	            actualHeadingElement = reviewsSection;
+	            extentInfoLog("Section is displayed: ", Action.isDisplayed(driver, actualHeadingElement));
+	            return;
+	        default:
+	            throw new IllegalArgumentException("Invalid section type provided: " + jumpToHeading);
+	    }
+
+	    String actualHeadingText = actualHeadingElement.getText();
+	    extentInfoLog("Section is displayed : ", Action.isDisplayed(driver, actualHeadingElement));
+	    Action.printAndAssert(actualHeadingText, sectionHeading);
+	    extentInfoLog(jumpToHeadingText + " jump to heading verified");
```

Comment:
@zahinSJI no need of this. remove this line.

Reply:
Removed